hack/local-up-karmada.sh

```diff
@@ -132,6 +132,8 @@ util::create_cluster "${PULL_MODE_CLUSTER_NAME}" "${PULL_MODE_CLUSTER_TMP_CONFIG
 export VERSION="latest"
 export REGISTRY="docker.io/karmada"
 make images GOOS="linux" --directory="${REPO_ROOT}"
+#clean up dangling images
+docker image prune -f
```

Comment:
What is the usage scenario of the dancing image?

Reply:
What about add `LABEL image-source=karmada` when docker build, so that we can only remove dangling images created by karmada by `docker image prune -f --filter="label=image-source=karmada"`

Reply:
> Excellent ideal! And furtherly, label it with a uuid, guarantee only clean up images made in this executaion.

I think there's no need to use uuid, just `LABEL maintainer="karmada.io"` should work fine. WDYT?